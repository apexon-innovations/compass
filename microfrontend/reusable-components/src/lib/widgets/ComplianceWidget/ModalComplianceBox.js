import React, { useState } from 'react'
import PropTypes from 'prop-types'
import { Tabs, Tab } from 'react-bootstrap'
import { Scrollbars } from 'react-custom-scrollbars'
import dayjs from 'dayjs'
import utc from 'dayjs/plugin/utc'

// Style
import style from './ComplianceWidget.module.scss'

dayjs.extend(utc)

const StatusCircle = ({ label, value, onClickCallBack, activeStatus, filterStatus }) => {
  return (
    <div
      className={[style.status, label === activeStatus ? style.active : ''].join(' ')}
      onClick={() => {
        onClickCallBack(label, filterStatus)
      }}
    >
      <div className={style.number}>
        <span className={style.span}>{value}</span>
      </div>
      <div className={style.label}>{label}</div>
    </div>
  )
}

const ModalBoxSection = ({ data }) => {
  return (
    <div className={style.taskCommentsWrapper}>
      {data.length > 0 ? (
        <ol>
          {' '}
          {data.map((item, index) => {
            return (
              <li key={index}>
                <div className={style.data}>
                  <a href={item.url} rel="noopener noreferrer" target="_blank">
                    {item.number} {item.name} ({item.type})
                  </a>
                </div>
                {item.ncDueDate ? (
                  <div className={style.dueDate}>
                    Due:{' '}
                    {dayjs(item.ncDueDate)
                      .utc()
                      .format('MM/DD/YYYY')}
                  </div>
                ) : (
                  ''
                )}
              </li>
            )
          })}{' '}
        </ol>
      ) : (
        <div className={style.noData}>Sorry, No Record Available.</div>
      )}
    </div>
  )
}

const filterTicketsByStatus = (data, filterStatus) => {
  const filterData = data.filter(item => {
    return filterStatus.indexOf(item.status) > -1
  })
  return filterData
}

const ModalComplianceBox = React.memo(({ response }) => {
  const [filterResponse, setFilterResponse] = useState({
    tickets: response.tickets,
    observations: response.observations,
    activeStatus: false,
  })
  const callBackSetActiveStatus = (status, filterStatus) => {
    if (filterResponse.activeStatus === status) {
      setFilterResponse({
        tickets: response.tickets,
        observations: response.observations,
        activeStatus: false,
      })
    } else {
      setFilterResponse({
        tickets: filterTicketsByStatus(response.tickets, filterStatus),
        observations: filterTicketsByStatus(response.observations, filterStatus),
        activeStatus: status,
      })
    }
  }

  return (
    <div className={style.modalComplianceBox}>
      {response && (
        <Tabs
          className="blackNWhite"
          defaultActiveKey="tabCompTickets"
          onSelect={() => callBackSetActiveStatus(filterResponse.activeStatus)}
        >
          <Tab eventKey="tabCompTickets" title="Tickets">
            <div className={style.workDetails}>
              <div className={style.taskStatus}>
                <StatusCircle
                  label={'To-Dos'}
                  value={response.complianceSummary.openTickets}
                  onClickCallBack={callBackSetActiveStatus}
                  activeStatus={filterResponse.activeStatus}
                  filterStatus={['Open', 'Reopened']}
                />
                <StatusCircle
                  label={'In-Progress'}
                  value={filterTicketsByStatus(response.tickets, ['In Progress']).length}
                  onClickCallBack={callBackSetActiveStatus}
                  activeStatus={filterResponse.activeStatus}
                  filterStatus={['In Progress']}
                />
                <StatusCircle
                  label={'Completed'}
                  value={response.complianceSummary.closedTickets}
                  onClickCallBack={callBackSetActiveStatus}
                  activeStatus={filterResponse.activeStatus}
                  filterStatus={['Closed', 'Resolved']}
                />
              </div>
              <div className={style.taskComments}>
                <Scrollbars autoHeight autoHeightMax={100}>
                  <ModalBoxSection data={filterResponse.tickets} />
                </Scrollbars>
              </div>
            </div>
          </Tab>
          <Tab eventKey="tabCompObservations" title="Observations">
            <div className={style.workDetails}>
              <div className={style.taskStatus}>
                <StatusCircle
                  label={'Open'}
                  value={response.complianceSummary.openObservations}
                  onClickCallBack={callBackSetActiveStatus}
                  activeStatus={filterResponse.activeStatus}
                  filterStatus={['Open', 'Reopened']}
                />
                <StatusCircle
                  label={'Addressed'}
                  value={response.complianceSummary.closedObservations}
                  onClickCallBack={callBackSetActiveStatus}
                  activeStatus={filterResponse.activeStatus}
                  filterStatus={['Closed', 'Resolved']}
                />
              </div>
              <div className={style.taskComments}>
                <Scrollbars autoHeight autoHeightMax={100}>
                  <ModalBoxSection data={filterResponse.observations} />
                </Scrollbars>
              </div>
            </div>
          </Tab>
        </Tabs>
      )}
    </div>
  )
})

StatusCircle.propTypes = {
  label: PropTypes.string,
  value: PropTypes.number,
  activeStatus: PropTypes.any,
  onClickCallBack: PropTypes.func,
  filterStatus: PropTypes.array,
}

ModalBoxSection.propTypes = {
  data: PropTypes.any,
}

ModalComplianceBox.propTypes = {
  response: PropTypes.any,
}
export default ModalComplianceBox
