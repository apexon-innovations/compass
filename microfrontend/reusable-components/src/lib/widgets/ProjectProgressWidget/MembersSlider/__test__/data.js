const data = {
  data: [
    {
      email: 'Pradeep.shrivastava@infostretch.com',
      name: 'Pradeep Shrivastava',
      dp: 'https://nest.infostretch.com/nativefw/images/profile_pic/2755.jpg?77098933',
    },
    {
      email: 'vijay.solanki@infostretch.com',
      name: 'Vijay Solanki',
      dp: 'https://nest.infostretch.com/nativefw/images/profile_pic/1546.jpg?9447621',
    },
    {
      email: 'suraj.bhosale@infostretch.com',
      name: 'Suraj Bhosale',
      dp: 'https://nest.infostretch.com/nativefw/images/profile_pic/3536.jpg?89267950',
    },
    {
      email: 'yatin.bhavsar@infostretch.com',
      name: 'Yatin Bhavsar',
      dp: 'https://nest.infostretch.com/nativefw/images/profile_pic/3104.jpg?9939973',
    },
    {
      email: 'vishal.kumar@infostretch.com',
      name: 'Vishal kumar',
      dp: 'https://nest.infostretch.com/nativefw/images/profile_pic/3057.jpg?72778930',
    },
    {
      email: 'gunjan.shah@infostretch.com',
      name: 'Gunjan Shah',
      dp: 'https://nest.infostretch.com/nativefw/images/profile_pic/2136.jpg?32186182',
    },
    {
      email: 'namra.porecha@infostretch.com',
      name: 'Namra Porecha',
      dp: 'https://nest.infostretch.com/nativefw/images/profile_pic/545.jpg?23733375',
    },
    {
      email: 'purushottam.Khandebharad@infostretch.com',
      name: 'Purushottam Khandebharad',
      dp: 'https://nest.infostretch.com/nativefw/images/profile_pic/3740.jpg?8948437',
    },
    {
      email: 'amit.ghadge@infostretch.com',
      name: 'Amit Ghadge',
      dp: 'https://nest.infostretch.com/nativefw/images/profile_pic/3855.JPG?34957269',
    },
  ],
}

export default data
