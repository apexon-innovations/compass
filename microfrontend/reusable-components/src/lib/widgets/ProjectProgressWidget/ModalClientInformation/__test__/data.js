const data = {
  data: {
    id: '5ee31002d48d514e801cb10b',
    name: 'Compass',
    clientName: 'Infostretch',
    initials: 'C',
    category: 'All (Dev & QA & Support)',
    iconLocation:
      'http://sampleicon.com/image.png',
    billingType: 'Internal',
    startDate: 'Wed Jan 01 00:00:00 UTC 2020',
    endDate: 'Thu Jan 02 00:00:00 UTC 2020',
    resources: {
      actual: 10,
      sow: 10,
    },
    healthMetrics: {
      overall: 'R',
      periodicReview: 'G',
      scope: 'A',
      deliverables: 'A',
      customer: 'G',
      schedule: 'R',
      financials: 'G',
      people: 'G',
      compliance: 'G',
      startUp: 'G',
      process: 'G',
      planning: 'G',
    },
    projectManager: [],
    deliveryManager: [
      {
        email: 'gunjan.shah@infostretch.com',
        name: 'Gunjan Shah',
        dp: 'https://nest.infostretch.com/nativefw/images/profile_pic/2136.jpg?32186182',
        nestProjects: ['Infostretch - DES'],
      },
    ],
    accountManager: [
      {
        email: 'gunjan.shah@infostretch.com',
        name: 'Gunjan Shah',
        dp: 'https://nest.infostretch.com/nativefw/images/profile_pic/2136.jpg?32186182',
        nestProjects: ['Infostretch - DES'],
      },
    ],
    teamMembers: [
      {
        email: 'Pradeep.shrivastava@infostretch.com',
        name: 'Pradeep Shrivastava',
        dp: 'https://nest.infostretch.com/nativefw/images/profile_pic/2755.jpg?77098933',
      },
      {
        email: 'vijay.solanki@infostretch.com',
        name: 'Vijay Solanki',
        dp: 'https://nest.infostretch.com/nativefw/images/profile_pic/1546.jpg?9447621',
      },
      {
        email: 'suraj.bhosale@infostretch.com',
        name: 'Suraj Bhosale',
        dp: 'https://nest.infostretch.com/nativefw/images/profile_pic/3536.jpg?89267950',
      },
      {
        email: 'yatin.bhavsar@infostretch.com',
        name: 'Yatin Bhavsar',
        dp: 'https://nest.infostretch.com/nativefw/images/profile_pic/3104.jpg?9939973',
      },
      {
        email: 'vishal.kumar@infostretch.com',
        name: 'Vishal kumar',
        dp: 'https://nest.infostretch.com/nativefw/images/profile_pic/3057.jpg?72778930',
      },
      {
        email: 'gunjan.shah@infostretch.com',
        name: 'Gunjan Shah',
        dp: 'https://nest.infostretch.com/nativefw/images/profile_pic/2136.jpg?32186182',
      },
      {
        email: 'namra.porecha@infostretch.com',
        name: 'Namra Porecha',
        dp: 'https://nest.infostretch.com/nativefw/images/profile_pic/545.jpg?23733375',
      },
      {
        email: 'purushottam.Khandebharad@infostretch.com',
        name: 'Purushottam Khandebharad',
        dp: 'https://nest.infostretch.com/nativefw/images/profile_pic/3740.jpg?8948437',
      },
      {
        email: 'amit.ghadge@infostretch.com',
        name: 'Amit Ghadge',
        dp: 'https://nest.infostretch.com/nativefw/images/profile_pic/3855.JPG?34957269',
      },
    ],
  },
  fileUploaderUrl:
    'https://compass-dev-api.infostretch.com/psr-service/project/5ee31002d48d514e801cb10b/logo',
}

export default data
