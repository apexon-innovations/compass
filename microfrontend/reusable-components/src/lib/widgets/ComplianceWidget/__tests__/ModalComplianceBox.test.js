/* eslint-disable */
import React from 'react'
import { render, unmountComponentAtNode } from 'react-dom'
import { act } from 'react-dom/test-utils'
import { sleep } from '../../../../__tests__/commonTestFunction'
import ModalComplianceBox from '../ModalComplianceBox'
import { modelComplianceData, noModelComplianceData } from './data'
const props = { ...modelComplianceData }

describe(`ModalComplianceBox`, () => {
  let container = null

  beforeEach(() => {
    // set up a DOM element as a render target
    container = document.createElement('div')
    document.body.appendChild(container)
  })

  afterEach(() => {
    // cleanup on exiting
    unmountComponentAtNode(container)
    container.remove()
    container = null
  })

  it(`Snapshot`, async () => {
    act(() => {
      render(<ModalComplianceBox {...{ response: modelComplianceData }} />, container)
    })
    await act(() => sleep(500))
    expect(container).toMatchSnapshot()
  })

  it(`Snapshot No data for project`, async () => {
    act(() => {
      render(<ModalComplianceBox {...{ response: noModelComplianceData }} />, container)
    })
    await act(() => sleep(500))
    expect(container).toMatchSnapshot()
  })
})
