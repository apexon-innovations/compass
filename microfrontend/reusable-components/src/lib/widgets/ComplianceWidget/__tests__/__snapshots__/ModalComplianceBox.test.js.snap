// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`ModalComplianceBox Snapshot 1`] = `
<div>
  <div>
    <ul
      class="blackNWhite nav nav-tabs"
      role="tablist"
    >
      <li
        class="nav-item"
        role="presentation"
      >
        <button
          aria-controls="react-aria6494588069-1-tabpane-tabCompTickets"
          aria-selected="true"
          class="nav-link active"
          data-rr-ui-event-key="tabCompTickets"
          id="react-aria6494588069-1-tab-tabCompTickets"
          role="tab"
          type="button"
        >
          Tickets
        </button>
      </li>
      <li
        class="nav-item"
        role="presentation"
      >
        <button
          aria-controls="react-aria6494588069-1-tabpane-tabCompObservations"
          aria-selected="false"
          class="nav-link"
          data-rr-ui-event-key="tabCompObservations"
          id="react-aria6494588069-1-tab-tabCompObservations"
          role="tab"
          tabindex="-1"
          type="button"
        >
          Observations
        </button>
      </li>
    </ul>
    <div
      class="tab-content"
    >
      <div
        aria-labelledby="react-aria6494588069-1-tab-tabCompTickets"
        class="fade tab-pane active show"
        id="react-aria6494588069-1-tabpane-tabCompTickets"
        role="tabpanel"
      >
        <div>
          <div>
            <div
              class=" "
            >
              <div>
                <span>
                  0
                </span>
              </div>
              <div>
                To-Dos
              </div>
            </div>
            <div
              class=" "
            >
              <div>
                <span>
                  0
                </span>
              </div>
              <div>
                In-Progress
              </div>
            </div>
            <div
              class=" "
            >
              <div>
                <span>
                  11
                </span>
              </div>
              <div>
                Completed
              </div>
            </div>
          </div>
          <div>
            <div
              style="position: relative; overflow: hidden; width: 100%; height: auto; min-height: 0; max-height: 100px;"
            >
              <div
                style="position: relative; overflow: scroll; margin-right: 0px; margin-bottom: 0px; min-height: 0; max-height: 100px;"
              >
                <div>
                  <ol>
                     
                    <li>
                      <div>
                        <a
                          href="https://jira.infostretch.com/browse/AUD-324"
                          rel="noopener noreferrer"
                          target="_blank"
                        >
                          AUD-324
                           
                          Training Plan for the team is available however same needs to be prepared in QMS Standard Template
                           (
                          Audit Finding
                          )
                        </a>
                      </div>
                      <div>
                        Due:
                         
                        08/02/2019
                      </div>
                    </li>
                    <li>
                      <div>
                        <a
                          href="https://jira.infostretch.com/browse/AUD-325"
                          rel="noopener noreferrer"
                          target="_blank"
                        >
                          AUD-325
                           
                          major Risk associated with this Project which needs to be identified and updated in the Risk Register
                           (
                          Audit Finding
                          )
                        </a>
                      </div>
                      <div>
                        Due:
                         
                        08/02/2019
                      </div>
                    </li>
                    <li>
                      <div>
                        <a
                          href="https://jira.infostretch.com/browse/AUD-324"
                          rel="noopener noreferrer"
                          target="_blank"
                        >
                          AUD-324
                           
                          Training Plan for the team is available however same needs to be prepared in QMS Standard Template
                           (
                          Audit Finding
                          )
                        </a>
                      </div>
                      <div>
                        Due:
                         
                        08/02/2019
                      </div>
                    </li>
                    <li>
                      <div>
                        <a
                          href="https://jira.infostretch.com/browse/AUD-325"
                          rel="noopener noreferrer"
                          target="_blank"
                        >
                          AUD-325
                           
                          major Risk associated with this Project which needs to be identified and updated in the Risk Register
                           (
                          Audit Finding
                          )
                        </a>
                      </div>
                      <div>
                        Due:
                         
                        08/02/2019
                      </div>
                    </li>
                    <li>
                      <div>
                        <a
                          href="https://jira.infostretch.com/browse/AUD-449"
                          rel="noopener noreferrer"
                          target="_blank"
                        >
                          AUD-449
                           
                          % Rework is continuously reported 0. Which is incorrect. It should be calculated by logging effort spent on bug fixing from testing and code reviews
                           (
                          Audit Finding
                          )
                        </a>
                      </div>
                      <div>
                        Due:
                         
                        07/20/2020
                      </div>
                    </li>
                    <li>
                      <div>
                        <a
                          href="https://jira.infostretch.com/browse/AUD-324"
                          rel="noopener noreferrer"
                          target="_blank"
                        >
                          AUD-324
                           
                          Training Plan for the team is available however same needs to be prepared in QMS Standard Template
                           (
                          Audit Finding
                          )
                        </a>
                      </div>
                      <div>
                        Due:
                         
                        08/02/2019
                      </div>
                    </li>
                    <li>
                      <div>
                        <a
                          href="https://jira.infostretch.com/browse/AUD-325"
                          rel="noopener noreferrer"
                          target="_blank"
                        >
                          AUD-325
                           
                          major Risk associated with this Project which needs to be identified and updated in the Risk Register
                           (
                          Audit Finding
                          )
                        </a>
                      </div>
                      <div>
                        Due:
                         
                        08/02/2019
                      </div>
                    </li>
                    <li>
                      <div>
                        <a
                          href="https://jira.infostretch.com/browse/AUD-449"
                          rel="noopener noreferrer"
                          target="_blank"
                        >
                          AUD-449
                           
                          % Rework is continuously reported 0. Which is incorrect. It should be calculated by logging effort spent on bug fixing from testing and code reviews
                           (
                          Audit Finding
                          )
                        </a>
                      </div>
                      <div>
                        Due:
                         
                        07/20/2020
                      </div>
                    </li>
                    <li>
                      <div>
                        <a
                          href="https://jira.infostretch.com/browse/AUD-324"
                          rel="noopener noreferrer"
                          target="_blank"
                        >
                          AUD-324
                           
                          Training Plan for the team is available however same needs to be prepared in QMS Standard Template
                           (
                          Audit Finding
                          )
                        </a>
                      </div>
                      <div>
                        Due:
                         
                        08/02/2019
                      </div>
                    </li>
                    <li>
                      <div>
                        <a
                          href="https://jira.infostretch.com/browse/AUD-325"
                          rel="noopener noreferrer"
                          target="_blank"
                        >
                          AUD-325
                           
                          major Risk associated with this Project which needs to be identified and updated in the Risk Register
                           (
                          Audit Finding
                          )
                        </a>
                      </div>
                      <div>
                        Due:
                         
                        08/02/2019
                      </div>
                    </li>
                    <li>
                      <div>
                        <a
                          href="https://jira.infostretch.com/browse/AUD-449"
                          rel="noopener noreferrer"
                          target="_blank"
                        >
                          AUD-449
                           
                          % Rework is continuously reported 0. Which is incorrect. It should be calculated by logging effort spent on bug fixing from testing and code reviews
                           (
                          Audit Finding
                          )
                        </a>
                      </div>
                      <div>
                        Due:
                         
                        07/20/2020
                      </div>
                    </li>
                     
                  </ol>
                </div>
              </div>
              <div
                style="position: absolute; height: 6px; display: none; right: 2px; bottom: 2px; left: 2px; border-radius: 3px;"
              >
                <div
                  style="position: relative; display: block; height: 100%; cursor: pointer; border-radius: inherit; background-color: rgba(0, 0, 0, 0.2);"
                />
              </div>
              <div
                style="position: absolute; width: 6px; display: none; right: 2px; bottom: 2px; top: 2px; border-radius: 3px;"
              >
                <div
                  style="position: relative; display: block; width: 100%; cursor: pointer; border-radius: inherit; background-color: rgba(0, 0, 0, 0.2);"
                />
              </div>
            </div>
          </div>
        </div>
      </div>
      <div
        aria-labelledby="react-aria6494588069-1-tab-tabCompObservations"
        class="fade tab-pane"
        id="react-aria6494588069-1-tabpane-tabCompObservations"
        role="tabpanel"
      >
        <div>
          <div>
            <div
              class=" "
            >
              <div>
                <span>
                  0
                </span>
              </div>
              <div>
                Open
              </div>
            </div>
            <div
              class=" "
            >
              <div>
                <span>
                  12
                </span>
              </div>
              <div>
                Addressed
              </div>
            </div>
          </div>
          <div>
            <div
              style="position: relative; overflow: hidden; width: 100%; height: auto; min-height: 0; max-height: 100px;"
            >
              <div
                style="position: relative; overflow: scroll; margin-right: 0px; margin-bottom: 0px; min-height: 0; max-height: 100px;"
              >
                <div>
                  <ol>
                     
                    <li>
                      <div>
                        <a
                          href="https://jira.infostretch.com/browse/AUD-444"
                          rel="noopener noreferrer"
                          target="_blank"
                        >
                          AUD-444
                           
                          There was an outstanding amount in invoices of approx 90K pounds
                           (
                          Audit Finding
                          )
                        </a>
                      </div>
                      <div>
                        Due:
                         
                        07/20/2020
                      </div>
                    </li>
                    <li>
                      <div>
                        <a
                          href="https://jira.infostretch.com/browse/AUD-445"
                          rel="noopener noreferrer"
                          target="_blank"
                        >
                          AUD-445
                           
                          5.	Identification of future training needs of team members was not done as the training register was not evident during the audit
                           (
                          Audit Finding
                          )
                        </a>
                      </div>
                      <div>
                        Due:
                         
                        07/20/2020
                      </div>
                    </li>
                    <li>
                      <div>
                        <a
                          href="https://jira.infostretch.com/browse/AUD-446"
                          rel="noopener noreferrer"
                          target="_blank"
                        >
                          AUD-446
                           
                          Risk Register - Risk no 1. Description and re assessement columns to be modified and risk to be closed. And add one risk related to Covid 19 and Data security threat at Project level.
                           (
                          Audit Finding
                          )
                        </a>
                      </div>
                      <div>
                        Due:
                         
                        07/20/2020
                      </div>
                    </li>
                    <li>
                      <div>
                        <a
                          href="https://jira.infostretch.com/browse/AUD-447"
                          rel="noopener noreferrer"
                          target="_blank"
                        >
                          AUD-447
                           
                          Communication Plan on confluence – Needs to be updated to give more clarity on the type of meetings,participants, the written communication,etc as explained during the audit
                           (
                          Audit Finding
                          )
                        </a>
                      </div>
                      <div>
                        Due:
                         
                        07/20/2020
                      </div>
                    </li>
                    <li>
                      <div>
                        <a
                          href="https://jira.infostretch.com/browse/AUD-444"
                          rel="noopener noreferrer"
                          target="_blank"
                        >
                          AUD-444
                           
                          There was an outstanding amount in invoices of approx 90K pounds
                           (
                          Audit Finding
                          )
                        </a>
                      </div>
                      <div>
                        Due:
                         
                        07/20/2020
                      </div>
                    </li>
                    <li>
                      <div>
                        <a
                          href="https://jira.infostretch.com/browse/AUD-445"
                          rel="noopener noreferrer"
                          target="_blank"
                        >
                          AUD-445
                           
                          5.	Identification of future training needs of team members was not done as the training register was not evident during the audit
                           (
                          Audit Finding
                          )
                        </a>
                      </div>
                      <div>
                        Due:
                         
                        07/20/2020
                      </div>
                    </li>
                    <li>
                      <div>
                        <a
                          href="https://jira.infostretch.com/browse/AUD-446"
                          rel="noopener noreferrer"
                          target="_blank"
                        >
                          AUD-446
                           
                          Risk Register - Risk no 1. Description and re assessement columns to be modified and risk to be closed. And add one risk related to Covid 19 and Data security threat at Project level.
                           (
                          Audit Finding
                          )
                        </a>
                      </div>
                      <div>
                        Due:
                         
                        07/20/2020
                      </div>
                    </li>
                    <li>
                      <div>
                        <a
                          href="https://jira.infostretch.com/browse/AUD-447"
                          rel="noopener noreferrer"
                          target="_blank"
                        >
                          AUD-447
                           
                          Communication Plan on confluence – Needs to be updated to give more clarity on the type of meetings
                           (
                          Audit Finding
                          )
                        </a>
                      </div>
                      <div>
                        Due:
                         
                        07/20/2020
                      </div>
                    </li>
                    <li>
                      <div>
                        <a
                          href="https://jira.infostretch.com/browse/AUD-444"
                          rel="noopener noreferrer"
                          target="_blank"
                        >
                          AUD-444
                           
                          There was an outstanding amount in invoices of approx 90K pounds
                           (
                          Audit Finding
                          )
                        </a>
                      </div>
                      <div>
                        Due:
                         
                        07/20/2020
                      </div>
                    </li>
                    <li>
                      <div>
                        <a
                          href="https://jira.infostretch.com/browse/AUD-445"
                          rel="noopener noreferrer"
                          target="_blank"
                        >
                          AUD-445
                           
                          5.	Identification of future training needs of team members was not done as the training register was not evident during the audit
                           (
                          Audit Finding
                          )
                        </a>
                      </div>
                      <div>
                        Due:
                         
                        07/20/2020
                      </div>
                    </li>
                    <li>
                      <div>
                        <a
                          href="https://jira.infostretch.com/browse/AUD-446"
                          rel="noopener noreferrer"
                          target="_blank"
                        >
                          AUD-446
                           
                          Risk Register - Risk no 1. Description and re assessement columns to be modified and risk to be closed. And add one risk related to Covid 19 and Data security threat at Project level.
                           (
                          Audit Finding
                          )
                        </a>
                      </div>
                      <div>
                        Due:
                         
                        07/20/2020
                      </div>
                    </li>
                    <li>
                      <div>
                        <a
                          href="https://jira.infostretch.com/browse/AUD-447"
                          rel="noopener noreferrer"
                          target="_blank"
                        >
                          AUD-447
                           
                          Communication Plan on confluence – Needs to be updated to give more clarity on the type of meetings
                           (
                          Audit Finding
                          )
                        </a>
                      </div>
                      <div>
                        Due:
                         
                        07/20/2020
                      </div>
                    </li>
                     
                  </ol>
                </div>
              </div>
              <div
                style="position: absolute; height: 6px; display: none; right: 2px; bottom: 2px; left: 2px; border-radius: 3px;"
              >
                <div
                  style="position: relative; display: block; height: 100%; cursor: pointer; border-radius: inherit; background-color: rgba(0, 0, 0, 0.2);"
                />
              </div>
              <div
                style="position: absolute; width: 6px; display: none; right: 2px; bottom: 2px; top: 2px; border-radius: 3px;"
              >
                <div
                  style="position: relative; display: block; width: 100%; cursor: pointer; border-radius: inherit; background-color: rgba(0, 0, 0, 0.2);"
                />
              </div>
            </div>
          </div>
        </div>
      </div>
    </div>
  </div>
</div>
`;

exports[`ModalComplianceBox Snapshot No data for project 1`] = `
<div>
  <div>
    <ul
      class="blackNWhite nav nav-tabs"
      role="tablist"
    >
      <li
        class="nav-item"
        role="presentation"
      >
        <button
          aria-controls="react-aria6494588069-2-tabpane-tabCompTickets"
          aria-selected="true"
          class="nav-link active"
          data-rr-ui-event-key="tabCompTickets"
          id="react-aria6494588069-2-tab-tabCompTickets"
          role="tab"
          type="button"
        >
          Tickets
        </button>
      </li>
      <li
        class="nav-item"
        role="presentation"
      >
        <button
          aria-controls="react-aria6494588069-2-tabpane-tabCompObservations"
          aria-selected="false"
          class="nav-link"
          data-rr-ui-event-key="tabCompObservations"
          id="react-aria6494588069-2-tab-tabCompObservations"
          role="tab"
          tabindex="-1"
          type="button"
        >
          Observations
        </button>
      </li>
    </ul>
    <div
      class="tab-content"
    >
      <div
        aria-labelledby="react-aria6494588069-2-tab-tabCompTickets"
        class="fade tab-pane active show"
        id="react-aria6494588069-2-tabpane-tabCompTickets"
        role="tabpanel"
      >
        <div>
          <div>
            <div
              class=" "
            >
              <div>
                <span>
                  0
                </span>
              </div>
              <div>
                To-Dos
              </div>
            </div>
            <div
              class=" "
            >
              <div>
                <span>
                  0
                </span>
              </div>
              <div>
                In-Progress
              </div>
            </div>
            <div
              class=" "
            >
              <div>
                <span>
                  0
                </span>
              </div>
              <div>
                Completed
              </div>
            </div>
          </div>
          <div>
            <div
              style="position: relative; overflow: hidden; width: 100%; height: auto; min-height: 0; max-height: 100px;"
            >
              <div
                style="position: relative; overflow: scroll; margin-right: 0px; margin-bottom: 0px; min-height: 0; max-height: 100px;"
              >
                <div>
                  <div>
                    Sorry, No Record Available.
                  </div>
                </div>
              </div>
              <div
                style="position: absolute; height: 6px; display: none; right: 2px; bottom: 2px; left: 2px; border-radius: 3px;"
              >
                <div
                  style="position: relative; display: block; height: 100%; cursor: pointer; border-radius: inherit; background-color: rgba(0, 0, 0, 0.2);"
                />
              </div>
              <div
                style="position: absolute; width: 6px; display: none; right: 2px; bottom: 2px; top: 2px; border-radius: 3px;"
              >
                <div
                  style="position: relative; display: block; width: 100%; cursor: pointer; border-radius: inherit; background-color: rgba(0, 0, 0, 0.2);"
                />
              </div>
            </div>
          </div>
        </div>
      </div>
      <div
        aria-labelledby="react-aria6494588069-2-tab-tabCompObservations"
        class="fade tab-pane"
        id="react-aria6494588069-2-tabpane-tabCompObservations"
        role="tabpanel"
      >
        <div>
          <div>
            <div
              class=" "
            >
              <div>
                <span>
                  0
                </span>
              </div>
              <div>
                Open
              </div>
            </div>
            <div
              class=" "
            >
              <div>
                <span>
                  0
                </span>
              </div>
              <div>
                Addressed
              </div>
            </div>
          </div>
          <div>
            <div
              style="position: relative; overflow: hidden; width: 100%; height: auto; min-height: 0; max-height: 100px;"
            >
              <div
                style="position: relative; overflow: scroll; margin-right: 0px; margin-bottom: 0px; min-height: 0; max-height: 100px;"
              >
                <div>
                  <div>
                    Sorry, No Record Available.
                  </div>
                </div>
              </div>
              <div
                style="position: absolute; height: 6px; display: none; right: 2px; bottom: 2px; left: 2px; border-radius: 3px;"
              >
                <div
                  style="position: relative; display: block; height: 100%; cursor: pointer; border-radius: inherit; background-color: rgba(0, 0, 0, 0.2);"
                />
              </div>
              <div
                style="position: absolute; width: 6px; display: none; right: 2px; bottom: 2px; top: 2px; border-radius: 3px;"
              >
                <div
                  style="position: relative; display: block; width: 100%; cursor: pointer; border-radius: inherit; background-color: rgba(0, 0, 0, 0.2);"
                />
              </div>
            </div>
          </div>
        </div>
      </div>
    </div>
  </div>
</div>
`;
